universal-application-tool-0.0.1/app/services/applicant/question/MultiSelectQuestion.java

```diff
@@ -61,6 +61,10 @@ public boolean hasValue() {
     return getSelectedOptionsValue().isPresent();
   }
 
+  public boolean isAnswered() {
```

Comment:
(blocking) Should this be added to the interface? or should there be an additional interface if `PresentsErrors` isn't the right scope?

It seems like if every `SomethingQuestion` has a public `isAnswered` method it should be available generically.

Reply:
Good point.... I think that sounds good. The name that comes to mind is `Answerable`. LMK if you think of something better.

Reply:
Hmm, I guess one awkward aspect of it is that multi-field question types, like `AddressQuestion` have multiple `Answered` methods, like `streetAnswered`, `cityAnswered`. We could have an overall `isAnswered` method that takes the union of those though.

I'm trying to think of what value it adds to have an interface and the overall `isAnswered` method. The only place we really use it is to determine whether or not to check for errors. With that in mind, maybe it does make the most sense to have `isAnswered` be part of the `PresentsErrors` interface.

Reply:
Updated. Let me know what you think of the change, @kendrickt .